packages/client/index.js

```diff
@@ -1,11 +1,3 @@
 const prisma = require('.prisma/client/index')
-const path = require('path')
 
 module.exports = prisma
-
-/**
```

Comment:
This is causing issues with the data proxy. We already generate annotations in the generated client for the other two engines. So this should not be needed anymore.